### PR TITLE
Update Uberon release workflow

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes https://github.com/obophenotype/uberon/issues/2544
@matentzn - I've created a secondary document for this, not sure if you want me to integrate with the ODK release workflow. Not sure if updating the ODK will erase all that too.
Happy to do either, lmk. Thanks!

### CODE REVIEW

File: docs/odk-workflows/ReleaseWorkflow.md [hunk 1]
```diff
@@ -21,11 +21,7 @@ Preparation:
 
 To actually run the release, you:
```
Comment (matentzn) [inline docs/odk-workflows/ReleaseWorkflow.md:23]: do not overwrite this document. it is automatically updated by ODK!

File: docs/uberon-release.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+✓ Authentication complete.
+- gh config set -h github.com git_protocol ssh
+✓ Configured git protocol
+✓ Uploaded the SSH key to your GitHub account: /Users/username/.ssh/id_ed25519.pub
+✓ Logged in as username
+```
+
+## Release Process
+
+1. Follow the preparation steps found in the [ODK-workflow release document](odk-workflows/ReleaseWorkflow.md)
```
Comment (matentzn) [inline docs/uberon-release.md:36]: probably needs fixing now - add everything here.

File: docs/uberon-release.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+1. Open a command line terminal window and navigate to the src/ontology directory (`cd uberon/src/ontology`)
+1. Run the release using `sh run.sh make uberon DEPLOY_GH=false`. This will build all files and copy them to the correct place. 
+1. Review the release as per the review release section in [ODK-workflow release document](odk-workflows/ReleaseWorkflow.md)
+1. Merge to main branch once reviewed and CI checks have passed
```
Comment (matentzn) [inline docs/uberon-release.md:40]: Make PR first!

Comment (matentzn) [reply docs/uberon-release.md:40]: Also get at least one core developer review!

File: docs/uberon-release.md (hunk 2 above)
Comment (shawntanzk) [reply docs/uberon-release.md]: I think for preparation it makes sense to copy the whole thing over, but for review theres a clear section so redirect would suffice

File: docs/uberon-release.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+1. Make sure that all changes to master are committed to Github (`git status` should say that there are no modified files)
+1. Locally make sure you have the latest changes from master (`git pull`)
+1. Checkout a new branch (e.g. `git checkout -b release-2021-01-01`)
+1. You may or may not want to refresh your imports as part of your release strategy (see [here](UpdateImports.md))(Note: in UBERON we decouple our imports and releases - we hence advice that you do not update imports)
+1. Make sure you have the latest ODK installed by running `docker pull obolibrary/odkfull`
+
+To actually run the release, you:
+
+1. Open a command line terminal window and navigate to the src/ontology directory (`cd uberon/src/ontology`)
+1. Run the release using `sh run.sh make IMP=false uberon DEPLOY_GH=false`. This will build all files and copy them to the correct place. (Note: the `IMP=false` is used to decouple imports refresh with release)
```
Comment (shawntanzk) [inline docs/uberon-release.md:50]: @matentzn I've explicitly put IMP=false to decouple imports with release, just wanted to make sure you are ok with this being in docs - figured given its internal uberon docs we can do this

Comment (matentzn) [reply docs/uberon-release.md:50]: No need - `uberon` already does that. I would remove it. Just `sh run.sh make uberon DEPLOY_GH=false`.

**Review by matentzn — APPROVED** [review]
Excellent, I would remove the IMP=false, add the uberon-release.md to mkdocs.yml and then this is good to merge. No need for a further review

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
no need to wait for qc

**Comment by shawntanzk** [comment]
I know it doesnt affect, but psychologically i get nervous without qc :p

**Pull request merged**